web/src/ui/shared/S3PathControl.tsx

```
@@ -0,0 +1,1670 @@
+        const collapsed: Crumb[] = [];
+        collapsed.push(crumbs[rootIndex]);
+
+        if (bucketIndex !== rootIndex) {
+            collapsed.push(crumbs[bucketIndex]);
+        }
+
+        if (shouldUseEllipsis) {
+            collapsed.push(ellipsisCrumb);
+        }
+
+        tailIndices.forEach(index => collapsed.push(crumbs[index]));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)